src/test/java/teammates/test/pageobjects/AppPage.java

```diff
@@ -216,6 +218,40 @@ public void waitForElementToDisappear(By by){
     }
     
     /**
+     * Waits for a list of elements to be invisible or not present, or timeout.
+     */
+    public void waitForElementsToDisappear(List<WebElement> elements) {
+        WebDriverWait wait = new WebDriverWait(browser.driver, TestProperties.inst().TEST_TIMEOUT);
+        wait.until(invisibilityOfAllElements(elements));
+    }
+    
+    /**
+     * Code adapted from SeleniumHQ's GitHub page.
+     * TODO to be removed when Selenium is upgraded to the version supporting this method.
```

Comment:
Disclaimer: not even the latest Selenium (2.52.0) has this method yet so upgrading now will not help.